card/lib/card/cache.rb

```diff
@@ -51,16 +29,18 @@ def restore
         prepopulate
       end
 
-      def reset_global # deprecated
-        reset_all
+      def reset_global
+        cache_by_class.each do |_klass, cache|
+          cache.soft.reset
+          cache.hard.annihilate if cache.hard
```

Comment:
"annihilate" is the new word for clearing the entire cache (and that of every site using the service).

This means reset_global and reset_all are now different.